app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
   end
 
+  def update?
+    is_himself?
+  end
+
   def is_himself?
-    current_user == record
+    record == current_user
```

Comment:
Yes, I agree. But once more, this PR is not to change the order of this. This PR is to solve another problem.

In my point of view it is only noise.